artifacthub/library/pod-security-policy/host-network-ports/1.1.0/suite.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Suite
+apiVersion: test.gatekeeper.sh/v1alpha1
+metadata:
+  name: host-network-ports
+tests:
+- name: use-of-host-networking-ports-blocked
+  template: template.yaml
+  constraint: samples/psp-host-network-ports/constraint.yaml
```

Comment:
We probably should add a hostNetwork test to detect bugs like the one highlighted below.

Reply:
yeah, added a the tests to `suite.yaml` for this.